docs/adding_packages/README.md

```diff
@@ -36,14 +36,16 @@ You can follow the three steps (:one: :two: :three:) described below! :tada:
 
 :one: The first step to add packages to ConanCenter is requesting access. To enroll in ConanCenter repository, please write a comment
 requesting access in this GitHub [issue](https://github.com/conan-io/conan-center-index/issues/4). Feel free to introduce yourself and
-your motivation to join ConanCenter.
+your motivation to join ConanCenter community.
 
-This process helps conan-center-index against spam and malicious code. The process is not not automated on purpose and the requests are generally approved on a weekly basis.
+This process helps ConanCenter against spam and malicious code. The process is not not automated on purpose and the requests are generally approved
```

Comment:
```suggestion
This process helps ConanCenter against spam and malicious code. The process is not automated on purpose and the requests are generally approved
```

Reply:
We actually manually look at each user and make sure they are legit 😆 it jsut the bot opens a PR so it's easy to see the list :)

The review is really manual